casper/src/main/resources/MultiSigRevVault.rho

```diff
@@ -107,9 +95,27 @@ in {
                     } |
 
                     contract verify(@auth, idxAuthPubKeysCh, unsealedCh) = {
-                      new unsealerTest in {
-                        // Check whether auth is in the list of authPubKeys
-                        @ListOps!("indexOf", authPubKeys, auth, *idxAuthPubKeysCh) |
```

Comment:
Just a question. The `equality` you mean is from this `indexOf` compare, right?

Reply:
Yes, exactly. Auth token are generated from public keys inside multi-sig contract for this comparison. Because this is not how they work they will always be different (unforgeable name created inside AuthKey contract) and authentication for multi-sig will never succeed.